### PR TITLE
Add `Marker` mock

### PR DESCRIPTION
**Pull request by Seggan**
# Description

Added `MarkerMock` and added it to `WorldMock`.

# Checklist
The following items should be checked before the pull request can be merged.
- [x] Code follows existing style.
- [x] Unit tests added (if applicable).

### CODE REVIEW

**Comment by Seggan** [comment]
I don't see error?

**Comment by Seggan** [comment]
Ah, `error: SignMock is not abstract and does not override abstract method getAllowedEditor() in Sign`. Should I implement this? Not really within the PR scope

**Comment by Thorinwasher** [comment]
> Ah, error: SignMock is not abstract and does not override abstract method getAllowedEditor() in Sign. Should I implement this? Not really within the PR scope

That should be in a separate PR

**Review by Thorinwasher — CHANGES_REQUESTED** [review]
Although Marker does not have a different interface than Entity, the behavior is a bit different. Here's the difference:

It's not possible for it to have any passengers; the Marker#addPassenger(Entity) method should not add any passengers to it and always return false.

(There's one more difference, but that is not implemented in EntityMock anyhow)

**Review by Thorinwasher — CHANGES_REQUESTED** [review]
I forgot to mention that there's a test in WorldMockTest which checks all spawnable entities (check for a huge Stream of EntityTypes and entity classes). Please add MarkerMock to that test.

File: src/test/java/be/seeseemelk/mockbukkit/WorldMockTest.java [hunk 1]
```diff
@@ -1330,7 +1331,8 @@ public static Stream<Arguments> getSpawnableEntities()
 				Arguments.of(EntityType.ZOMBIFIED_PIGLIN, PigZombieMock.class),
 				Arguments.of(EntityType.TRIDENT, Trident.class),
 				Arguments.of(EntityType.SPECTRAL_ARROW, SpectralArrow.class),
-				Arguments.of(EntityType.ARROW, Arrow.class)
+				Arguments.of(EntityType.ARROW, Arrow.class),
+				Arguments.of(EntityType.MARKER, MarkerMock.class)
```
Comment (Seggan) [inline src/test/java/be/seeseemelk/mockbukkit/WorldMockTest.java:1335]: @Thorinwasher already done

**Review by Thorinwasher — APPROVED** [review]
Looks good to me!

I missed that you already added the test there, sorry about that.

**Pull request merged**